File: 045_Haskell/fizzbuzz.hs
-- This is a comment.

module Main where

fizzBuzz :: Int -> String
fizzBuzz i =
    case (i `mod` 3, i `mod` 5) of
        (0, 0) -> "FizzBuzz"
        (0, _) -> "Fizz"
        (_, 0) -> "Buzz"
        _ -> show i

main :: IO ()
main = mapM_ (putStrLn . fizzBuzz) [1 .. 100]

File: 051_Clojure/fizzbuzz.clj
; This is a comment.

;Documentation: |https://clojuredocs.org/|

(defn fizzbuzz [i]
    (cond
        (and (= 0 (mod i 3)) (= 0 (mod i 5))) "FizzBuzz"
        (= 0 (mod i 3)) "Fizz"
        (= 0 (mod i 5)) "Buzz"
        :else i
    )
)

(dotimes [i 100]
    (println (fizzbuzz (+ i 1)))
)

